src/main/java/org/mastodon/mamut/treesimilarity/NodeMappings.java

```diff
@@ -0,0 +1,130 @@
+package org.mastodon.mamut.treesimilarity;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.mastodon.mamut.treesimilarity.tree.Tree;
+
+/**
+ * Utility class for {@link ZhangUnorderedTreeEditDistance} that provides
+ * static factory methods for the easy creation of {@link NodeMapping}s.
+ */
+class NodeMappings
+{
+	private NodeMappings()
+	{
+		// prevent from instantiation
+	}
+
+	/**
+	 * @return An empty {@link NodeMapping} with the specified cost.
+	 * (Please note that the costs for an empty mapping are almost never
+	 * zero!)
+	 */
+	public static < T > NodeMapping< T > empty( double cost )
```

Comment:
I think, it would be more explicit, if EmptyNodeMapping was returned by this method.